tests/util/benchmark.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
```

Comment:
If you don't need any bash-specific extensions using /bin/sh is probably more desirable.